packages/pulsar-updater/src/main.js

```diff
@@ -0,0 +1,240 @@
+const { CompositeDisposable } = require("atom");
+
+let shell;
+let superagent;
+let findInstallMethod;
+
+class PulsarUpdater {
+  activate() {
+    this.disposables = new CompositeDisposable();
+    this.cache = require("./cache.js");
+    this.manuallyTriggeredCheck = false;
+
+    this.disposables.add(
+      atom.commands.add("atom-workspace", {
+        "pulsar-updater:check-for-update": () => {
+          this.manuallyTriggeredCheck = true;
+          this.checkForUpdates();
```

Comment:
Can we do something like

```suggestion
          this.checkForUpdates({ manual: true });
```

instead? Then we don't have to worry about resetting `manuallyTriggeredCheck` later on.